### ISSUE TITLE
Desktop client doesn't work on macOS Catalina

### ISSUE THREAD

**Issue by 4Luke4** [smogon/pokemon-showdown#6485]
Pretty much what the title states.

The [Desktop Client](https://pokemonshowdown.com/autodownload/mac) doesn't work on this OS, I cannot start any match, keeps saying "Loading format"...

**Comment by Zarel** [comment]
I'm using it fine in Catalina. This is most likely a problem with your internet connection.

**Comment by 4Luke4** [comment]
> I'm using it fine in Catalina. This is most likely a problem with your internet connection.

No, you're wrong. My internet connection works fine (I get email notifications, can use the Browser version of Showdown and the like). But for some reason this Desktop Client doesn't work (see the attached screenshot).

<img width="269" alt="Showdown" src="https://user-images.githubusercontent.com/39967126/77413297-65a95a00-6dbf-11ea-98a3-933efe20d5a4.png">

**Comment by Zarel** [comment]
Look, whatever problem you're having, it's clearly not purely a Catalina desktop client issue, because I can't reproduce this:

![image](https://user-images.githubusercontent.com/551184/77413416-8bf9d600-6d7c-11ea-9f64-aded9ea79e46.png)

Are you on the old desktop client? Have you tried reinstalling?

**Comment by Zarel** [comment]
If you still can't get this to work, we can try TeamViewer and I can see if there's something I can do about it. Find me on Discord; I'm Zarel#1111

**Comment by 4Luke4** [comment]
> Are you on the old desktop client?

I'm using the one from the link I posted above. Could you post the link to the most recent version here?

**Comment by Zarel** [comment]
The link above is the most recent version.

**Comment by Zarel** [comment]
But did you redownload it today?

**Comment by 4Luke4** [comment]
> But did you redownload it today?

Yes, still nothing though....

**Comment by 4Luke4** [comment]
@Zarel 

To clarify: I can log in, watch other battles, type some text in the chat (lobby)...
I cannot start any kind of match nor open Teambuilder (_lol zarel this is a horrible teambuilder
that's because we're not done loading it..._)

**Comment by Zarel** [comment]
Yes, that's very clearly a loading error. If it works with Chrome, it's not an internet error. Contact me on Discord! I want to use screenshare to figure this one out.

**Comment by 4Luke4** [comment]
> If it works with Chrome...

I use Safari, not Chrome...

**Comment by Zarel** [comment]
Do you not have Discord? You could also DM me on Twitter, I'm https://twitter.com/PokemonShowdown

Or if you don't have anything, e-mail me, zarel@pokemonshowdown.com

**Comment by HoeenCoder** [comment]
Possibly the websocket is blocked on your network? This looks familiar to another issue: https://www.smogon.com/forums/threads/technical-support-v2-read-original-post-before-posting.3469929/page-46#post-8381057

If thats the case its not anything we can fix unfortunately.

**Comment by Zarel** [comment]
If it works on Safari but not the desktop client, it's way subtler than just network blocking. I want to figure it out, but again, I do need some form of screen sharing to do it.

**Comment by 4Luke4** [comment]
> Or if you don't have anything, e-mail me, [zarel@pokemonshowdown.com](mailto:zarel@pokemonshowdown.com)

OK, so what should I do exactly?

**Comment by Zarel** [comment]
On second thought, apparently Apple's locked down TeamViewer a lot. Let's do iMessage screen sharing.

Can you open Messages in your Mac, and send an iMessage to `zarelsl@gmail.com`?

**Comment by Zarel** [comment]
We figured out, it was because PS doesn't work without access to the Documents folder, and Catalina no longer grants that automatically.
